R/cal-apply.R

```diff
@@ -0,0 +1,98 @@
+#---------------------------------- Apply --------------------------------------
+
+#' Applies a calibration to a set of prediction probabilities
+#' @details It currently supports data.frames only. It extracts the `truth` and
+#' the estimate columns names, and levels, from the calibration object.
+#' @param .data An object that can process a calibration object.
+#' @param calibration The calibration object (`cal_object`).
+#' @param ... Optional arguments; currently unused.
+#' @examples
+#' w_calibration <- cal_estimate_logistic(segment_logistic, Class)
+#'
+#' cal_apply(segment_logistic, w_calibration)
+#' @export
+cal_apply <- function(.data, calibration, ...) {
+  UseMethod("cal_apply")
+}
+
```

Comment:
I will end up getting the `.data` and calibration object order confused. 

Can we add a methods for the calibration objects that errors with a message about the order of the arguments.

Reply:
I agree!! I was gonna change it before sending over the PR, and it totally left my mind. I plan to use `x`